src/sumoLogger.js

```diff
@@ -59,20 +60,16 @@ function sendLogs() {
     }, function (error, response) {
       var err = !!error || response.statusCode < 200 || response.statusCode >= 400;
 
-      if (err && currentConfig.hasOwnProperty('onError')) {
```

Comment:
No need for these checks as onError will always either be provided by the user or the `NOOP` function